Segment-ComScore/Classes/SEGComScoreIntegration.m

```diff
@@ -34,20 +34,24 @@ - (instancetype)initWithSettings:(NSDictionary *)settings andComScore:(id)scorAn
         SCORPublisherConfiguration *config = [SCORPublisherConfiguration publisherConfigurationWithBuilderBlock:^(SCORPublisherConfigurationBuilder *builder) {
             // publisherId is also known as c2 value
             builder.publisherId = settings[@"c2"];
-            builder.publisherSecret = settings[@"publisherSecret"];
-            builder.applicationName = settings[@"appName:"];
-            builder.usagePropertiesAutoUpdateInterval = [settings[@"autoUpdateInterval"] integerValue];
-            builder.secureTransmission = [(NSNumber *)[self.settings objectForKey:@"useHTTPS"] boolValue];
-            builder.liveTransmissionMode = SCORLiveTransmissionModeLan;
-
-            if ([(NSNumber *)[self.settings objectForKey:@"autoUpdate"] boolValue] && [(NSNumber *)[self.settings objectForKey:@"foregroundOnly"] boolValue]) {
-                builder.usagePropertiesAutoUpdateMode = SCORUsagePropertiesAutoUpdateModeForegroundOnly;
-            } else if ([(NSNumber *)[self.settings objectForKey:@"autoUpdate"] boolValue]) {
-                builder.usagePropertiesAutoUpdateMode = SCORUsagePropertiesAutoUpdateModeForegroundAndBackground;
-            } else {
-                builder.usagePropertiesAutoUpdateMode = SCORUsagePropertiesAutoUpdateModeDisabled;
-            }
+            builder.secureTransmissionEnabled = [(NSNumber *)[self.settings objectForKey:@"useHTTPS"] boolValue];
         }];
+        
+        SCORAnalytics.configuration.applicationName = settings[@"appName:"];
```

Comment:
Looks like [per the docs](https://app.box.com/s/csd2t9sb9z0m6gdadg852ysl4724jac8/file/545670443958) they are looking for `[SCORAnalyticsSCORAnalytics configuration].applicationName = value;`. Curious if this works the same?

Reply:
Yes, it does. :)